subprojects/packagefiles/spdk-win/patches/spdk-win-0002-lib-nvme-restore-spdk_nvme_ctrlr_get_registers.patch

```diff
@@ -0,0 +1,130 @@
+From d287ca5a625568c5f4b77d07431e45e14016e253 Mon Sep 17 00:00:00 2001
```

Comment:
We already have a file with prefix `spdk-win-0002` perhaps this should be renamed to `spdk-win-0003`

Reply:
Right, renamed.